### PR TITLE
Advertise Vero4K as a Mali platform

### PR DESCRIPTION
**Pull request by hissingshark**
Addresses Issue #2943 more appropriately.

### CODE REVIEW

**Comment by joolswills** [comment]
Thanks - Everything works ok ? there's still some vero4k logic in some modules - maybe some of those cases can be simplified also ?

File: scriptmodules/system.sh [hunk 1]
```diff
@@ -441,5 +441,5 @@ function platform_vero4k() {
     __default_cflags="-I/opt/vero3/include -L/opt/vero3/lib -O2 -mcpu=cortex-a7 -mfpu=neon-vfpv4 -mfloat-abi=hard -ftree-vectorize -funsafe-math-optimizations"
     __default_asflags=""
     __default_makeflags="-j4"
-    __platform_flags="arm armv7 neon vero4k gles"
+    __platform_flags="arm armv7 neon mali vero4k gles"
```
Comment (joolswills) [inline scriptmodules/system.sh]: You can remove vero4k from platform flags btw as it's automatically included.

Comment (hissingshark) [reply scriptmodules/system.sh]: Good point.

**Comment by hissingshark** [comment]
Some of the vero4k logic is to trigger platform specific flags that were added upstream, but I'll see if there's anything else more generic I can handle in `helpers.sh`, to unclutter the individual modules.

**Comment by joolswills** [comment]
Thanks

**Pull request merged**